### PR TITLE
Add Codecov to CI workflow

### PR DESCRIPTION
**Pull request by svillegas-cdd**
> Codecov is the all-in-one code coverage reporting solution for any
> test suite — giving developers actionable insights to deploy reliable
> code with confidence.

- Codecov web site: https://about.codecov.io/
- Documentation: https://docs.codecov.com/

Ref: https://cordada.aha.io/requirements/TECHINFRA-194-9
Ref: https://cordada.aha.io/features/TECHINFRA-194

### CODE REVIEW

**Comment by fpinto-cdd** [comment]
@svillegas-cdd Can you check why the coverage files are not found? https://github.com/fyntex/drf-pagination-utils/actions/runs/5060426903/jobs/9083311035?pr=126

**Comment by svillegas-cdd** [comment]
> @svillegas-cdd Can you check why the coverage files are not found? https://github.com/fyntex/drf-pagination-utils/actions/runs/5060426903/jobs/9083311035?pr=126

I haven't found why these file are not find, @jtrobles-cdd could you help me?

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -116,6 +116,12 @@ jobs:
           source "$PYTHON_VIRTUALENV_ACTIVATE"
           make test-coverage-report
 
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3.1.2
+        with:
+          directory: ./test-reports/coverage/
```
Comment (jtrobles-cdd) [inline .github/workflows/ci.yaml:122]:
```suggestion
          directory: ./test_reports/coverage/
```

File: setup.cfg [hunk 2]
```diff
@@ -40,3 +40,6 @@ where = .
 include =
   fyntex
 exclude =
+
+[coverage:xml]
+output = test-reports/coverage/xml/coverage.xml
```
Comment (jtrobles-cdd) [inline setup.cfg:45]: Move this to `.coveragerc.test.ini` and change it to:

```suggestion
[xml]
output = test_reports/coverage/xml/coverage.xml
```

**Review by jtrobles-cdd — CHANGES_REQUESTED** [review]

**Comment by jtrobles-cdd** [comment]
> > @svillegas-cdd Can you check why the coverage files are not found? https://github.com/fyntex/drf-pagination-utils/actions/runs/5060426903/jobs/9083311035?pr=126
> 
> I haven't found why these file are not find, @jtrobles-cdd could you help me?

See https://github.com/fyntex/drf-pagination-utils/pull/126#pullrequestreview-1442841577.

**Review by fpinto-cdd — APPROVED** [review]

File: .coveragerc.test.ini [hunk 3]
```diff
@@ -17,3 +17,6 @@ show_missing = True
 
 [html]
 directory = test_reports/coverage/html
+
+[xml]
+output = test_reports/coverage/xml/coverage.xml
```
Comment (jtrobles-cdd) [inline .coveragerc.test.ini:22]: Can you move this between `[report]` and `[html]`?

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -116,6 +116,12 @@ jobs:
           source "$PYTHON_VIRTUALENV_ACTIVATE"
           make test-coverage-report
 
+      - name: Upload coverage reports to Codecov
+        uses: codecov/codecov-action@v3.1.2
```
Comment (jtrobles-cdd) [inline .github/workflows/ci.yaml:120]:
```suggestion
        uses: codecov/codecov-action@v3.1.4
```

File: README.md [hunk 5]
```diff
@@ -15,6 +15,11 @@ REST Framework.
 | `master` | Production | [GitHub](https://github.com/fyntex/drf-pagination-utils/tree/master)  | [![GitHub Actions](https://github.com/fyntex/drf-pagination-utils/actions/workflows/ci-cd.yaml/badge.svg?branch=master)](https://github.com/fyntex/drf-pagination-utils/actions/workflows/ci-cd.yaml?query=branch:master) |
 
 
+| Code Coverage                                                                                                                                     |
+|---------------------------------------------------------------------------------------------------------------------------------------------------|
```
Comment (jtrobles-cdd) [inline README.md:19]:
```suggestion
| Code Coverage |
| ------------- |
```

File: README.md [hunk 6]
```diff
@@ -15,6 +15,11 @@ REST Framework.
...
+| [![Codecov](https://codecov.io/gh/fyntex/drf-pagination-utils/branch/develop/graph/badge.svg)](https://codecov.io/gh/fyntex/drf-pagination-utils) |
```
Comment (jtrobles-cdd) [inline README.md:20]:
```suggestion
| [![Codecov](https://codecov.io/gh/fyntex/drf-pagination-utils/branch/develop/graph/badge.svg?token=66RN4HVW2C)](https://codecov.io/gh/fyntex/drf-pagination-utils) |
```

**Review by jtrobles-cdd — CHANGES_REQUESTED** [review]
- [ ] URL of Aha requirement is for a different VCS repository. Please fix PR description and commit messages.

**Review by jtrobles-cdd — APPROVED** [review]

**Pull request merged**